modules/ROOT/pages/design-mocking-service.adoc

```diff
@@ -22,37 +27,49 @@ baseUri: https://anypoint.mulesoft.com/mocking/api/v1/links/d0422e05-e779-4016-9
 # baseUri: http://jsonplaceholder.typicode.com
 ----
 
-You can run the mocking service from the text editor, the visual editor, and from Anypoint Exchange. If you want to run the mocking service from Anypoint Exchange after publishing your API specification, see "To Simulate Data in REST APIs" in the Anypoint Exchange documentation. A link to that topic is in the *See also* section at the end of this topic.
+You can run the mocking service from the text editor, the visual editor, and from Anypoint Exchange. See xref:exchange::ex2-to-simulate-api-data.adoc[Simulate Data in REST APIs] for information on running the mocking service from Anypoint Exchange after publishing your API specification.
 
 [NOTE]
 ====
-API Designer's public, mocked URI is also visible in Exchange. There, people who discover your API can try it out by using the built-in Exchange mocking service, or by using the public URI either in their code or in a tool for testing API implementations.
+API Designer's public, mocked URI is also visible from Exchange. There, people who discover your API can try it out by using the built-in Exchange mocking service, or by using the public URI either in their code or in a tool for testing API implementations.
 ====
 
 == Procedure
 
-* If you are using the text editor in API Designer:
+* Enable the mocking service.
```

Comment:
Again, this action does not enable the mocking service but creates a public link. Users can use mocking service without this step